web/annotation_layer_builder.css

```diff
@@ -237,6 +250,10 @@
   width: 100%;
 }
 
+.annotationLayer .fileAttachmentAnnotation .popupTriggerArea {
+  position: absolute;
+}
+
```

Comment:
In addition to the FileAttachmentAnnotation case, the `popupTriggerArea`-class may also be used as a fallback (see below).
So, should we perhaps just move the `position: absolute;` into the CSS rule *just above* instead?

https://github.com/mozilla/pdf.js/blob/652a6df62bdc6a1765f64714d96b9454433f847f/src/display/annotation_layer.js#L454-L459

Reply:
I tried that and some popups are moving in the reftests. From my pov it doesn't matter that much but I wanted to minimize the change. I'll push and updated patch and we'll see what's the best.

Reply:
Thanks for testing that!

Looking at the results, we probably want limit this to just the FileAttachmentAnnotation to reduce "unrelated" changes in this patch.